httpapi/userapi/bootstrapper.go

```diff
@@ -0,0 +1,29 @@
+package userapi
+
+import (
+	"github.com/centrifuge/go-centrifuge/documents"
+	"github.com/centrifuge/go-centrifuge/errors"
+	"github.com/centrifuge/go-centrifuge/extensions/transferdetails"
+)
+
+// BootstrappedUserAPIService key maps to the Service implementation in Bootstrap context.
+const BootstrappedUserAPIService = "UserAPI Service"
+
+// Bootstrapper implements bootstrap.Bootstrapper.
+type Bootstrapper struct{}
+
+// Bootstrap adds transaction.Repository into context.
+func (b Bootstrapper) Bootstrap(ctx map[string]interface{}) error {
+	docSrv, ok := ctx[documents.BootstrappedDocumentService].(documents.Service)
+	if !ok {
+		return errors.New("failed to get %s", documents.BootstrappedDocumentService)
+	}
+
+	tdSrv, ok := ctx[transferdetails.BootstrappedTransferDetailService].(transferdetails.Service)
+	if !ok {
+		return errors.New("failed to get %s", transferdetails.BootstrappedTransferDetailService)
+	}
+
+	ctx[BootstrappedUserAPIService] = Service{docSrv, tdSrv}
```

Comment:
why is `UserAPI` allowed to use `documents.Service` ? for me this defeats the purpose of core vs user separation. It's one of the main problems I solved in my pr.

Reply:
i think the question for me here is why not use the docSrv as wrapped in the coreAPISrv instead?

Reply:
Ahh. missed this one. Thanks!